### PR TITLE
Arjav Changes for the edit and add screens

### PR DESCRIPTION
**Pull request by aarjavshah**

### CODE REVIEW

File: src/components/AddProfile.js [hunk 1]
```diff
@@ -187,15 +188,20 @@ class AddProfile extends React.Component {
                           label="Yes"
                           type="radio"
                           name={`isReferral`}
-                          onChange={e => this.setState({ referred: true })}
+                          onChange={e => {this.setState({referred: true});
```
Comment (CGIBIFE) [inline src/components/AddProfile.js]: you can combine all setState together :
this.setState({referred:true, disabledRefferedBy:false})

File: src/components/AddProfile.js [hunk 2]
```diff
@@ -187,15 +188,20 @@ class AddProfile extends React.Component {
...
                         />
                         <Form.Check
                           inline
                           required
                           label="No"
                           type="radio"
                           name={`isReferral`}
-                          onChange={e => this.setState({ referred: false })}
+                          onChange={e => {this.setState({referred: false});
+                            this.setState({referredBy:null});
```
Comment (CGIBIFE) [inline src/components/AddProfile.js]: you can combine all setState together :

Comment (aarjavshah) [reply src/components/AddProfile.js]: Updated.

File: src/components/AddProfile.js (hunk 1 above)
Comment (aarjavshah) [reply src/components/AddProfile.js]: updated

File: src/components/AddProfile.js [hunk 3]
```diff
@@ -207,9 +210,10 @@ class AddProfile extends React.Component {
                       </Form.Label>
                       <Col sm="10">
                         <Form.Control
-                          required
+                          required = {(this.state.disabledRefferedBy)? "true" : "false"}
```
Comment (CGIBIFE) [inline src/components/AddProfile.js]: I think true and false should be boolean

File: src/components/ProfilesTable.js [hunk 4]
```diff
@@ -573,14 +589,14 @@ class CustomTable extends React.Component {
                         <tr>
                           <td>Referred by</td>
                           <td>
-                            <Form.Group controlId="profileStatus" as={Row}>
+                          <Form.Group controlId="referredBy" as={Row}>
                               <Form.Control
+                                required = {(this.state.disabledRefferedBy)? "true" : "false"}
```
Comment (CGIBIFE) [inline src/components/ProfilesTable.js]: Same can be Boolean

Comment (aarjavshah) [reply src/components/ProfilesTable.js]: fixed

File: src/components/AddProfile.js (hunk 3 above)
Comment (aarjavshah) [reply src/components/AddProfile.js]: fixed

File: src/components/AddProfile.js [hunk 5]
```diff
@@ -207,9 +210,10 @@ class AddProfile extends React.Component {
                       </Form.Label>
                       <Col sm="10">
                         <Form.Control
-                          required
+                          required = {(this.state.disabledRefferedBy)? true : false}
                           type="text"
                           placeholder="Enter Name"
+                          disabled = {(this.state.disabledRefferedBy)? "disabled" : ""}
```
Comment (CGIBIFE) [inline src/components/AddProfile.js]: I think disabled string can be from constant!

**Review by CGIBIFE — APPROVED** [review]

**Pull request merged**